ansible/all-in-one/roles/xsnippet/files/nginx.conf

```diff
@@ -0,0 +1,32 @@
+load_module modules/ngx_http_js_module.so;
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
```

Comment:
nit: just curious, what's up with these comments? tcp_nopush and gzip?

Reply:
No clue... This is the default config that is shipped with the nginx Docker image. The two things I changed are:

* added `load_module modules/ngx_http_js_module.so;` at the very top
* set `worker_processes  1;` as we discussed before

I can clean up the comments.